AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -414,8 +414,8 @@ m.n11.com##.modal-backdrop
 fastweb.it###stdCookie
 bursadabugun.com###_re_cookie_policy
 zacks.com##.disclosure-fixed-slab[aria-label="Cookie Consent"]
-psd-nuernberg.de##.lightbox--cookie-consent
-psd-nuernberg.de##.darken-layer
+vrhs.de,psd-nuernberg.de##.lightbox--cookie-consent
+vrhs.de,psd-nuernberg.de##.darken-layer
```

Comment:
These rules are blocking page scroll. You should exchange them.
1. Add exception to the `cookies_whitelist.txt`.
```
! https://github.com/AdguardTeam/AdguardFilters/issues/71072
vrhs.de#@#.cookie-consent:not(body):not(html)
```
2. Add the rule `vrhs.de#%#AG_onLoad(function() {if (!document.cookie.includes("cookieConsent")) {var g = new MutationObserver(function() {var b = document.querySelector('.cookie-consent .cookie-consent__settings ~ button[type="submit"]');b && b.click();});g.observe(document, {childList: !0,subtree: !0});setTimeout(function() {g.disconnect()}, 1E4)}});` to the `cookies_specific.txt` near similar rules `! JS rules`.

Reply:
Thanks for help

Reply:
You are welcome.